src/main/java/com/wanted/preonboarding/ticket/application/performance/controller/PerformanceController.java

```diff
@@ -0,0 +1,45 @@
+package com.wanted.preonboarding.ticket.application.performance.controller;
+
+import com.wanted.preonboarding.core.domain.response.ResponseHandler;
+import com.wanted.preonboarding.ticket.application.aop.annotation.ExecutionTimer;
+import com.wanted.preonboarding.ticket.application.performance.service.PerformanceService;
+import com.wanted.preonboarding.ticket.domain.dto.response.PerformanceDetail;
+import com.wanted.preonboarding.ticket.domain.dto.response.PerformanceInfo;
+import com.wanted.preonboarding.ticket.domain.dto.request.RegisterPerformance;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@ExecutionTimer
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/performance")
+public class PerformanceController {
+
+    private final PerformanceService performanceService;
+
+    @GetMapping("/list")
+    public ResponseEntity<ResponseHandler<List<PerformanceInfo>>> getAllPerformanceInfoList() {
+        return performanceService.getAllPerformanceInfoList();
+    }
+
+    @GetMapping("/detail/{id}/{round}")
```

Comment:
한번 고민할법한 포인트 조심스럽게 코멘트 달아드려요
저희가 리소스를 중심으로 URI를 설계하는 디자인하는 REST URI에는 전달하기에는 모호하다고 생각이 들어요.
영상/상세/id/round 라고 하면 상세의 식별값인지 영상의 식별값인지? 식별값의 키??인지?
[참조할만한 링크](https://learn.microsoft.com/ko-kr/azure/architecture/best-practices/api-design#define-api-operations-in-terms-of-http-methods)

아래는 참고의 예시이고 성호님이 생각하시는 더 좋은 디자인이 있으시면 좋을꺼 같습니다!

CASE 1
```suggestion
    @GetMapping("{id}/detail/?round={round}")
```
CASE 2
```suggestion
    @GetMapping("{id}/detail/{디테일ID}/rounds/{round}")
```

Reply:
감사합니다. 조금 더 명확하게 표현 가능한 디자인을 고민해보도록 하겠습니다..!! 😊